keps/sig-api-machinery/4006-transition-spdy-to-websockets/README.md

```diff
@@ -645,38 +637,24 @@ in back-to-back releases.
 
 #### Alpha
 
+- Implement the alpha version of the `RemoteCommand` subprotocol, and surface the new
+  `kubectl exec`, `kubectl cp`, and `kubectl attach` behind a `kubectl` environment
+  variable which is **OFF** by default.
+- Implement the alpha version of the `PortForward` subprotocol, and surface the new
+  `kubectl port-forward` behind a `kubectl` environment variable which is **OFF** by default.
 - `WebSocketExecutor` and `FallbackExecutor` completed and functional behind the `kubectl`
   environment variable KUBECTL_REMOTE_COMMAND_WEBSOCKETS which is **OFF** by default.
 - `StreamTranslatorProxy` successfully integrated into the `UpgradeAwareProxy`
   behind an API Server feature flag which is off by default.
-- Initial unit tests completed and enabled.
-- Initial integration tests completed and enabled.
-- Initial e2e tests completed and enabled.
+- Initial `exec`, `cp`, `attach`, and `port-forward` unit tests completed and enabled.
```

Comment:
it's confusing to call out port-forward tests and alpha version of the `PortForward` subprotocol here as requirements for alpha when the port-forward parts of the design are marked as TBD